doc/cephadm/troubleshooting.rst

```diff
@@ -176,3 +176,39 @@ To access the admin socket, first enter the daemon container on the host::
 
     [root@mon1 ~]# cephadm enter --name <daemon-name>
     [ceph: root@mon1 /]# ceph --admin-daemon /var/run/ceph/ceph-<daemon-name>.asok config show
+
+Manually deploying a MGR daemon
```

Comment:
In which cases this procedure is needed?

Reply:
if you end up with a cluster without MGRs.

Reply:
fixed